tests/common/request.py

```
@@ -29,19 +29,24 @@ def payload(function_name: str, *args) -> dict:
-    host: str = os.environ["RPCHOST"],
-    port: str = os.environ["RPCPORT"],
+    protocol: str,
+    host: str,
+    port: str,
 ):
+    jsonrpc_url = (
+        protocol + "://" + host + ":" + port + "/api"
+        if protocol and host and port
+        else os.environ.get("JSONRPC_SERVER_URL", "http://localhost:4000")
+    )
     return requests.post(
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)